data-service-backend/target/classes/api.yaml

```diff
@@ -0,0 +1,405 @@
+openapi: 3.0.3

```

Comment:
What are the files in data-service-backend for? I thought these folders are not used anymore?